tools/fileinfo/features/pe-loader-corruptions/test.py

```diff
@@ -492,3 +492,35 @@ def test_corrupted_pe(self):
         self.assertEqual(self.fileinfo.output["loaderError"]["code"], 43)
         self.assertEqual(self.fileinfo.output["loaderError"]["code_text"], 'LDR_ERROR_ENTRY_POINT_ZEROED')
         self.assertEqual(self.fileinfo.output["loaderError"]["loadable_anyway"], 'true')
+
+class Test044(Test):
```

Comment:
This PR adds the following two files:
```
tools/fileinfo/features/pe-loader-corruptions/000-correct-file-signed-32bit.ex_
tools/fileinfo/features/pe-loader-corruptions/000-correct-file-signed-64bit.ex_
```
Shouldn't we add them to `Test000` that checks valid files? Without adding them to the test, they won't be used. This is probably just an oversight.

Reply:
Originally I wanted them there as original (correct) files, but actually why not.
https://github.com/avast/retdec-regression-tests/pull/43/commits/985e670a93ba84c8a04c8f1aa455b65933d821e5